Examples/Modules/qed/schwinger/analysis_schwinger.py

```diff
@@ -0,0 +1,104 @@
+#! /usr/bin/env python
+
+# Copyright 2020 Luca Fedeli, Neil Zaim
+#
+# This file is part of WarpX.
+#
+# License: BSD-3-Clause-LBNL
+
+## This module is imported by each of the scripts used to analyze the Schwinger tests.
+
+## The pair production rate is calculated using the formula described in
+## Bulanov, S. S., et al. Physical review letters 104.22 (2010): 220404.
+
+import yt
+import numpy as np
+import sys
+import re
+
+# define some parameters
+
+c = 299792458.
+m_e = 9.10938356e-31
+e = 1.6021766208e-19
+hbar = 1.054571800e-34
+E_S = m_e**2*c**3/e/hbar # Schwinger field
+
+dV = (1.e-6)**3 # total simulation volume
+dt = 2.407291502e-16
+filename = sys.argv[1]
+
+Ex_test = 0.
+Ey_test = 0.
+Ez_test = 0.
+Bx_test = 0.
+By_test = 0.
+Bz_test = 0.
+
+# Find which test we are doing
+test_number = re.search( 'qed_schwinger([1234])', filename ).group(1)
+if test_number == '1':
+    Ex_test = 1.e16
+    Bx_test = 16792888.570516706
+    By_test = 5256650.141557486
+    Bz_test = 18363530.799561853
+elif test_number == '2':
+    Ex_test = 1.e18
+    Bx_test = 1679288857.0516706
+    By_test = 525665014.1557486
+    Bz_test = 1836353079.9561853
+elif test_number == '3':
+    Ey_test = 1.0321239524474501e+17
+elif test_number == '4':
+    Ez_test = 2.5e+20
+    By_test = 833910154604.3563
+else:
+    assert(False)
```

Comment:
Could you add a quick description for each test? e.g.,
```py
if test_number == '1':
    // Low-field test, no particle should be created
```
(or whatever comment is relevant)